doc/output-details-and-examples/src/output-files/eplusout-eio.tex

```diff
@@ -1205,29 +1248,84 @@ \subsubsection{Field: End-Use Category}\label{field-end-use-category}
 
 This field shows the end-use category for this lights statement. Usage can be reported by end-use category.
 
-\subsubsection{Field: Nominal Minimum Lighting Level \{W\}}\label{field-nominal-minimum-lighting-level-w}
+\subsubsection{Field: Minimum Lighting Level for All Day Types \{W\}}\label{field-minimum-lighting-level-for-all-day-types-w}
+
+This numeric field is the calculated minimum amount of lighting in Watts based on the calculated lighting level (above) * the minimum value for the lights schedule across all day types (weekday, weekends, holidays, design days and custom days). It may be useful in diagnosing errors that occur during simulation.
 
-This numeric field is the calculated minimum amount of lighting in Watts based on the calculated lighting level (above) * the minimum value (annual) for the lights schedule. It may be useful in diagnosing errors that occur during simulation.
+\subsubsection{Field: Maximum Lighting Level for All Day Types \{W\}}\label{field-maximum-lighting-level-for-all-day-types-w}
 
-\subsubsection{Field: Nominal Maximum Lighting Level \{W\}}\label{field-nominal-maximum-lighting-level-w}
+This numeric field is the calculated maximum amount of lighting in Watts based on the calculated lighting level (above) * the maximum value for the lights schedule across all day types (weekday, weekends, holidays, design days and custom days). It may be useful in diagnosing errors that occur during simulation.
 
-This numeric field is the calculated maximum amount of lighting in Watts based on the calculated lighting level (above) * the maximum value (annual) for the lights schedule. It may be useful in diagnosing errors that occur during simulation.
+\subsubsection{Field: Minimum Lighting Level for Weekdays \{W\}}\label{field-minimum-lighting-level-for-weekdays-w}
+
+This numeric field is the calculated minimum amount of lighting in Watts based on the calculated lighting level (above) * the minimum value for the lights schedule for weekday day types (Monday through Friday).
```

Comment:
Noting that the wording from here forward is different from the description of people.

    based on the number of people field

    based on the calculated lighting level (above)